explainer-server/app/views/explainEditor.scala.html

```diff
@@ -68,6 +66,24 @@
             }
         }
 
+        /*
+         * This function (and the accompanying handler) exists because there doesn't seem to exist a way
+         * to generate a checkbox using ScalaTags (ScalaJS's HTML generation library)
+         */
+        function installDisplayTypeTickBox(displayType){
+            console.log(displayType);
+            var checkedstring = displayType=='Expandable' ? ' checked ' : ''
+            $(".explainer-editor__displayType-inner").html('<input class="explainer-editor__displayType-checkbox" type="checkbox" name="displayType" value="long" '+checkedstring+'> Long explainer')

```

Comment:
@shtukas - Just to clarify on the classes point. It's 100% best practice to use classes for everything in CSS, in JS it's more of a "why not?" kind of case. The uniqueness denoted by an ID should only apply to the HTML. For JS you're better off using the data attributes or selecting some other way (creating the element in JS and binding it there, for example).